### PR TITLE
CASM-3747 support ARM / docker multi-arch builds

### PR DESCRIPTION
**Pull request by jpdavis-prof**
### Summary and Scope

Build arm64 docker images using docker multi-platform builds. The arm64 images are built in x86 runners, so arm64 runners are not needed.

https://jira-pro.its.hpecorp.net:8443/browse/CASM-3747

#### Issue Type

- RFE Pull Request

### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
 
 
### Risks and Mitigations
Low risk.

### CODE REVIEW

**Comment by rustydb** [comment]
nit: please include the JIRA in the commit's subject.

File: Makefile [hunk 1]
```diff
@@ -30,8 +32,10 @@ endif
 all: image
 
 image:
-	docker build --secret id=SLES_REGISTRATION_CODE --pull ${DOCKER_ARGS} --tag '${NAME}:${VERSION}' .
-	docker tag '${NAME}:${VERSION}' ${NAME}:${VERSION}-${TIMESTAMP}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}-${VERSION}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}-${VERSION}-${TIMESTAMP}
+	docker buildx create --use
+	docker buildx build --platform=linux/amd64,linux/arm64 --secret id=SLES_REGISTRATION_CODE --pull ${DOCKER_ARGS} .
```
Comment (rustydb) [inline Makefile:36]: I'm not familiar with the `buildx build` action here, is this adding both platforms in parallel or is it building one after the other (sequentially)?

Comment (jpdavis-prof) [reply Makefile:36]: It builds both ARCH's simultaneously.

**Review by manderson-hpe — APPROVED** [review]

**Pull request merged**